### DISCUSSION TITLE
Any ideas how to get Twitter (X) feed working?

### DISCUSSION THREAD

**Discussion by mdkberry** [Athou/commafeed#1137] [Q&A]
I am using the below, and it was working but I think Twitter is blocking access. Currently I get sporadic feeds but mostly they go red. I wondered if anyone has figured out a workaround for getting Twitter feeds:

Feed URL: 
https://rsshub.app/twitter/user/username/readable=1&authorNameBold=1&showQuotedAuthorAvatarInDesc=1&showEmojiForRetweetAndReply=1&addLinkForPics=1&showTimestampInDescription=1&showQuotedInTitle=1&heightOfPics=150

Results in...

_"Unable to refresh feed https://rsshub.app/twitter/user/username/readable=1&authorNameBold=1&showQuotedAuthorAvatarInDesc=1&showEmojiForRetweetAndReply=1&addLinkForPics=1&showTimestampInDescription=1&showQuotedInTitle=1&heightOfPics=150 : status code: 404, reason phrase: Server returned HTTP error code 404"_

&

_"Generate an API key in your profile first."_

I tried the latter, and it didnt work, though I was having issues with other sites but recently updated to the latest commafeed and that solved all but the Twitter issue.

Anyone got ideas for workarounds?

**Comment by CPUGPU** [comment]
Just a guess, but as it appears you are using the demo instance of RSSHub, the feed may be running into a rate limit as all twitter requests would come from that one IP.   You may want to check the route on the RSSHub itself.

From [RSSHub's FAQ](https://docs.rsshub.app/en/faq.html):

**Q: Can I use the demo instance？**

**A:** rsshub.app (opens new window) is the demo instance provided, running the latest build of RSSHub from master branch, the cache is set 120 minutes and it's free to use. However, if you see an badge **strict anti-crawler policy** for route, this means popular websites such as Facebook etc. may pose a request quota on individual IP address, which means it can get unreliable from time to time for the demo instance. You are encouraged to host your own RSSHub instance to get a better usability.
